diff --git a/dist/index.js b/dist/index.js
index 808983e10b00a8ec5dde97e2f7a1c99208cc7da8..43605f3a90a50d8a2c2a1aee62875e2f538a949d 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1236,7 +1236,7 @@ var SlackAdapter = class _SlackAdapter {
       this.logger.warn("Chat instance not initialized, ignoring event");
       return;
     }
-    if (event.subtype && event.subtype !== "bot_message") {
+    if (event.subtype && event.subtype !== "bot_message" && event.subtype !== "file_share") {
       this.logger.debug("Ignoring message subtype", {
         subtype: event.subtype
       });
